youtube_dl/extractor/canvas.py

```diff
@@ -53,6 +53,15 @@ def _real_extract(self, url):
                     'url': format_url,
                 })
         self._sort_formats(formats)
+        
+        subtitles = {}
+        for target in data['subtitleUrls']:
```

Comment:
Today it's there - tomorrow it's not. Extractors must be as robust as possible and tolerate to layout changes. Subtitles is not a critical metadata to extract thus when `subtitleUrls` disappears (if ever) if must not break the main extraction routine.